Add deleting-confirmation-flag document (#595)
longhorn/website: content/docs/1.4.0/references/settings.md

The default instance manager image used by the manager. Can be changed on the manager starting command line only.

#### Deleting Confirmation Flag

> This flag is designed to prevent Longhorn from being accidentally uninstalled which will lead to data lost.

Reviewer: ```suggestion
This flag protects Longhorn from unexpected uninstallation which leads to data loss.
```

Author: Thank you. Fixed

Set this flag to **true** to allow Longhorn uninstallation.

> If this flag **false**, Longhorn uninstallation job will fail.

Reviewer: ```suggestion
If this flag is **false**, the Longhorn uninstallation job will fail.
```

Author: Thank you. Fixed
